docs/user/dashboard/dashboard.asciidoc

```diff
@@ -4,505 +4,186 @@
 
-. Change the options, then click *Close*.
+* When the read-only indicator appears, you have insufficient privileges
+to create or save dashboards. The options to create and save dashboards are not visible. For more information,
+refer to <<xpack-security-authorization,Granting access to {kib}>>.
 
 [float]
-[[add-layers-and-indices]]
-==== Add layers and indices
+[[create-the-dashboard]]
+== Create dashboards
 
-To compare and analyze data from different sources, you can visualize multiple data layers and indices. Multiple layers and indices are
-supported in area, line, and bar charts.
+Open the dashboard, then open the visualization builder. 
 
-To add a layer, click *+*, then drag and drop the data fields for the new layer.
+. Open the main menu, click *Dashboard*.
 
-[role="screenshot"]
-image::images/lens_layers.png[Add layers]
-
-To view a different index, click the index name in the editor, then select a new one.
-
-[role="screenshot"]
-image::images/lens_index_pattern.png[Add index pattern]
-
-Ready to try out *Lens*? Refer to the <<lens-tutorial,step-by-step tutorial>>.
-
-[float]
-[[tsvb]]
-=== Create panels with TSVB
-
-*TSVB* is a time series data visualizer that allows you to use the full power of the Elasticsearch aggregation framework. To use *TSVB*,
-you can combine an infinite number of <<aggregation-reference,aggregations>> to display your data.
-
-With *TSVB*, you can:
-
-* Create visualizations, data tables, and markdown panels.
-* Create visualizations with multiple indices.
-* Change the aggregation and labels to customize the data.
+. To create a dashboard, click *Create dashboard*.
 +
-[role="screenshot"]
-image::images/tsvb.png[TSVB UI]
-
-[float]
-[[configure-the-data]]
-==== Configure the data
-
-With *TSVB*, you can add and display multiple data sets to compare and analyze. {kib} uses many types of <<aggregation-reference,aggregations>> that you can use to build
-complex summaries of that data.
+To open an existing dashboard, click the dashboard you want to view.
 
-. Select *Data*. If you are using *Table*, select *Columns*.
-. From the *Aggregation* drop down, select the aggregation you want to visualize.
+. To create a panel, click *Create panel*, then click *Lens* on the *New visualization* window.
 +
-If you don’t see any data, change the <<set-time-filter,time filter>>.
-+
-To add multiple aggregations, click *+*.
-. From the *Group by* drop down, select how you want to group or split the data.
-. To add another data set, click *+*.
-+
-When you have more than one aggregation, the last value is displayed, which is indicated by the eye icon.
+To add a saved panel, click *Add from library*, then select the panel you want to add. When a panel contains a stored query, both queries are applied.
 
 [float]
-[[change-the-data-display]]
-==== Change the data display
-
-To find the best way to display your data, *TSVB* supports several types of panels and charts.
+[[create-the-visualization]]
+== Create visualizations
```

Comment:
This section is 100% duplicate info from the Lens end-to-end guide which is on the same page: can we delete it?